libs/extensions/src/NanoDimuonVertex.cpp

```diff
@@ -68,14 +68,17 @@ float NanoDimuonVertex::GetCollinearityAngle() {
   return ptVector.DeltaPhi(Lxyz);
 }
 
-float NanoDimuonVertex::GetPATpTLxyDPhi() {
-  std::string category = GetVertexCategory();
-  if(category == "PatDSA" && !isDSAMuon1() && isDSAMuon2()) {
-    auto muonFourVector = asNanoMuon(muon1)->GetFourVector();
-    TVector3 ptVector(muonFourVector.Px(), muonFourVector.Py(), muonFourVector.Pz());
-    return ptVector.DeltaPhi(Lxyz);
+float NanoDimuonVertex::GetMuonpTLxyDPhi(int muonIndex) {
+  std::shared_ptr<PhysicsObject> muon;
+  if (muonIndex == 1) muon = muon1;
+  else if (muonIndex == 2) muon = muon2;
```

Comment:
Why do we count from 1 and not from 0?

Reply:
Actually I've always named the muons in the vertex fit as muon1 and muon2, so here I thought it confusing to give input 1 and get return value for muon 2

Reply:
Alright.